Main/Main.js

```diff
@@ -1,25 +1,73 @@
+// React
+import React, { useState, useEffect } from "react";
 import { View, StyleSheet } from "react-native";
+// Components
 import FromContainer from "./From/FromContainer.js";
 import ToContainer from "./To/ToContainer.js";
+import FlatButton from "../Global/Button"
+// Utils
+import { API_URL, API_KEY } from "@env";
+import CountryAndCurrency from "@workmate/country-and-currency";
+
+
+const Main = () => {
+
+  const [fromCountry, setFromCountry] = useState("");
+  const [fromCountryFlag, onChangefromCountryFlag] = useState("");
+  const [fromAmount, onChangefromAmount] = useState("");
+  const [fromCurrency, onChangefromCurrency] = useState("");
+
+  const [toCountry, setToCountry] = useState("");
+  const [toCountryFlag, onChangetoCountryFlag] = useState("");
+  const [toAmount, onChangetoAmount] = useState("");
+  const [toCurrency, onChangetoCurrency] = useState("");
+
+
+  const uppercaseWords = (str) =>
```

Comment:
I guess if we are uppercasing every letter, there is no need for this function anymore

Reply:
I think we still need it cuz uppercasing every letter is just for display.
The country name that user enters should be only first letter uppercase as that is what library wants which this function you made works for so I am not sure if we can get rid of it..
It was just a suggestion for design, I tried few things and country in uppercase looked good.. 😂